docs/05. Strategies.md

```diff
@@ -108,15 +108,15 @@ return [
 ## Creating custom strategies
 
 Creating a custom strategy is rather easy. Let's say we want to create a strategy that integrates with
-the [ApiProblem](https://github.com/zfcampus/zf-api-problem) Zend Framework 2 module:
+the [ApiProblem](https://github.com/laminas-api-tools/api-tools-api-problem) Laminas Framework 2 module:
```

Comment:
Another instance of "Laminas Framework 2" that could probably just be abbreviated to Laminas.